### ISSUE TITLE
Adjustable brightness levels and Moodlight defaults

### ISSUE THREAD

**Issue by nagyrobi** [dgomes/hasp-lvgl-custom-component#12]
Would be nice to have these configurable from yaml, as more plates can be present in the system and they may require different settings:
- a service to set the `idle_brightness` and `awake_brightness` dynamically at HA runtime. To use with automations which could monitor light conditions in the room.
- a default color for moodlight (specific to that room, for example) + service to change it
- an option to switch moodlight on before turning screen off (now possible with automation, but can be triggered only after the screen turns off thus the unit blinks through full-dark for a moment, see below)
- plates's short and long idle times (they are configurable form plate's web interface but would look nice to read them from there and set them from HA)

**Comment by nagyrobi** [comment]
Currently workink moodlight switcher automations, which would work better if handled by component:

```
- alias: lanbon-moodlight-idle
  initial_state: 'on'
  trigger:
    - platform: state
      entity_id: sensor.plate_idle
      from: 'short'
      to: 'long'
  action:
    - service: mqtt.publish
      data:
        topic: hasp/plate/command/page
        payload: 1
    - service: mqtt.publish
      data:
        topic: hasp/plate/command/moodlight
        payload: '{"state":"ON","color":{"r":255,"g":0,"b":255}}'

- alias: lanbon-moodlight-idle-not
  initial_state: 'on'
  trigger:
    - platform: state
      entity_id: sensor.plate_idle
      from: 'long'
      to: 'off'
  action:
    - service: mqtt.publish
      data:
        topic: hasp/plate/command/moodlight
        payload: '{"state":"OFF","color":{"r":255,"g":0,"b":255}}'

```

**Comment by dgomes** [comment]
I will definitely implement moodlight as a HA light entity with all the associated services.

**Comment by nagyrobi** [comment]
Autodiscovery with 0.4.0-dev already does this for moodlight and backlight.

**Comment by nagyrobi** [comment]
But! Moodlight colors are forgotten between restarts.

**Comment by nagyrobi** [comment]
Tested with new component version:
- turn on moodlight from HA entity (the one created by this component)
- select a nice color
- turn the moodlight on and off repeatedly from HA - everything's fine
- reboot plate, wait for LWT to refresh the device
- check moodlight entity: color forgotten.

**Comment by dgomes** [comment]
please check the latest release

**Comment by nagyrobi** [comment]
Color still forgotten by the component after power-cycling the plate.
